quay-enterprise/bitbucket-app.md

```diff
@@ -0,0 +1,19 @@
+# Creating an OAuth Application in BitBucket
+
+- Log into BitBucket
```

Comment:
if the opening line suggested is added, I don't think we need more expansion here